system/handlers/admin/emailCenter/Logs.cfc

```diff
@@ -50,6 +55,27 @@ component extends="preside.system.base.AdminHandler" {
 		);
 	}
 
+	public string function resendEmail( event, rc, prc ) {
```

Comment:
Picky, but we should rename this to `resendEmailAction` + we should ensure that it creates an audit log entry.